conftest.py

```diff
@@ -35,7 +36,18 @@ def create_database_connections():
 
 @pytest.fixture
 def sample_dag():
-    yield DAG("test_dag", default_args={"owner": "airflow", "start_date": DEFAULT_DATE})
+    try:
+        repo = git.Repo(search_parent_directories=True)
+        sha = repo.head.object.hexsha
```

Comment:
@dimberman, what are we gaining by using the Git hash instead of adding a `uuid.uuid4()` to the DAG name, which has more chances of being unique?